docs/_static/css/light.css

```
@@ -0,0 +1,31 @@
+/* links */
+.rst-content a{
+    color: #002fa7;
+}
+
+.rst-content a:visited {
+    color: #002fa7;
+}
+
+.rst-content .highlighted {
+	background: #eac545;
+}
+
+/* code blocks highlights */
+.rst-content pre {
+	background: #e5e5e5;
+}
+
+.wy-side-nav-search img{
+	width: 80px !important;
+}
+
+/* version sidebar */
+.wy-side-nav-search .version {
+	color: #272525 !important;
+}
+
+/* title color */
```

"title color" sounds as if these were titles in the text
```suggestion
/* menu section headers */
```

---

@4molybdenum2 Can you fix this one too?